src/elements/header/header/header.scss

```diff
@@ -105,6 +105,12 @@
   margin-inline-start: var(--sbb-header-first-item-margin-inline-start);
 }
 
+::slotted(:is(sbb-header-button, sbb-header-link)) {
```

Comment:
Am I right, that the header actions should not have an own size property?

Reply:
From what i saw on Figma only the height changes, so i don't feel the need for another property.

Reply:
If this is true, I agree. Simple solution :)

Reply:
the actions should have a size property: (m/s):
https://www.figma.com/design/9r6xSfNmEfCFxl1yFYedrj/Lyne-Components?node-id=29161-1064932&node-type=FRAME&t=5aYk7fIu5xvRs0Xu-11
sorry for the bad documentation in figma. it is now fixed.